bin/bootstrap

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -x
+
+# this hook is no longer needed, so we're removing it
+rm -f .git/hooks/pre-push
```

Comment:
This seems a bit aggressive if someone has their own hooks, no? I guess based on feedback it probably doesn't matter much either way

Reply:
Fair point; I'll remove it.